package NAlice.NWonderlogs;

import "alice/wonderlogs/protos/uniproxy_prepared.proto";

import "mapreduce/yt/interface/protos/extension.proto";

message TUniproxyPreparedWrapper {
    option (NYT.default_field_flags) = SERIALIZATION_YT;

    optional string Uuid = 1 [json_name = "uuid", (NYT.key_column_name) = "uuid"];
    optional string MessageId = 2 [json_name = "message_id", (NYT.key_column_name) = "message_id"];
    optional TUniproxyPrepared Value = 3 [json_name = "value", (NYT.column_name) = "value"];
}
